BackofficeBundle/Resources/public/coffee/DisplayApiErrorView.coffee

```diff
@@ -2,15 +2,29 @@ DisplayApiErrorView = OrchestraView.extend(
   initialize: (options) ->
     @options = @reduceOption(options, [
       'errors'
-      'domContainer'
     ])
-    @loadTemplates [
-        'OpenOrchestraBackofficeBundle:BackOffice:Underscore/apiError'
-    ]
+    @initializer options
     return
 
-  render: ->
-    @setElement @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/apiError', @options)
-    @options.domContainer.append @$el
+  initializer: (options) ->
+    errors = options.errors
+    if (typeof errors.error != 'undefined')
+      launchNotification errors.error, 'error'
+    else
+      for error of errors
+        launchNotification error, 'warning', error
     return
+
+  launchNotification = (error, type) ->
```

Comment:
If you put the launchNotification in the class, you should use `@launchNotificaton` to run it.

I would rather put the method in the orchestra libs, this way, it will be reusable in the rest of the project